src/fe-gtk/palette.c

```diff
@@ -37,7 +37,7 @@
 #include "../common/typedef.h"
 
 
-GdkColor colors[] = {
+GdkColor colors[MAX_COL+1] = {
```

Comment:
For be clear with other developers, comment the array index for each item and add the constants in the positions where they start, also add the constants of particular colors like XTEXT_MARK_FG